packages/commonwealth/client/scripts/views/pages/NotificationSettings/index.tsx

```diff
@@ -121,18 +118,15 @@ const NotificationSettings = () => {
               Get updates on new threads and discussions from these communities
             </CWText>
 
-            {Object.entries(bundledSubs)
-              .sort((x, y) => x[0].localeCompare(y[0]))
-              .map(([communityName]) => {
-                const communityInfo = app?.config.chains.getById(communityName);
-                return (
-                  <CommunityEntry
-                    key={communityInfo.id}
-                    communityInfo={communityInfo}
-                    communityAlert={communityAlertsIndex[communityInfo.id]}
-                  />
-                );
-              })}
+            {app.user.addresses.map((current) => {
```

Comment:
`app.user.addresses` will have duplicate community entries, in cases when multiple addresses are connected to a single community. Lets try to get unique addresses with `getUniqueUserAddresses()` from `packages/commonwealth/client/scripts/helpers/user.ts` and then use this

Reply:
Thanks! I updated the PR to get the unique addresses but also sort them.  getUniqueUserAddresses only returns the address strings on .address but I need to get the original AddressInfo objects back to get the community.  This is done with one function. I could reuse this code and then re-join but it with the original index but it would be about the same number of lines of code.